### ISSUE TITLE
Should SCONE architecture encompass a device hot-spotting to the UE?

### ISSUE THREAD

**Issue by SpencerDawkins** [mjoras/SCONE-PROTOCL#32]
@danwing asked about a device hot-spotting to the UE, in a comment on #27. I should ask more broadly,

- would this affect the SCONEPRO reference architecture?
- if so, do we think hot-spotting is in scope for SCONEPRO? 
- If so, does support for hot-spotting need to be mentioned in the charter?

**Comment by wesley-eddy** [comment]
Yes, the signaling must work in this case of hot-spotting, as well as related cases where the UE is not the end-host running the client application, such as the case of fixed wireless access, for instance.  I plan to suggest a couple of sentences to Matt for the requirements document to make it clear, and can update this with a link to the PR.   I don't think it bears special mention for the charter, necessarily.

**Comment by wesley-eddy** [comment]
Intended to be resolved by: https://github.com/mjoras/sadcdn-video-optimization-requirements/pull/1
